src/fsharp/FSharp.Build/Fsc.fs

```diff
@@ -120,8 +120,12 @@ type FscCommandLineBuilder () =
 //The goal is to have the most common/important flags available via the Fsc class, and the
 //rest can be "backdoored" through the .OtherFlags property.
 
-type [<Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")>] Fsc() as this = 
-    inherit ToolTask()
+type
+    [<Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly")>]
```

Comment:
You can place the attribute before the ``type`` keyword, it looks much nicer,

Reply:
well that's the whole point of cleanup ... consider it done :-)